Lib/test/test_shutil.py

```diff
@@ -579,6 +579,32 @@ def _filter(src, names):
             shutil.rmtree(src_dir)
             shutil.rmtree(os.path.dirname(dst_dir))
 
+    def test_copytree_arg_types_of_ignore(self):
+        join = os.path.join
+        exists = os.path.exists
+
+        src_dir = self.mkdtemp()
+        dst_dir = join(self.mkdtemp(), 'destination')
```

Comment:
you can use `self.addCleanup(shutil.rmtree, src_dir)` and save the try/finally block